handbook/business-operations/README.md

```diff
@@ -159,6 +159,32 @@ Once notified, BizOps takes the following steps:
 - Create an [offboarding issue](https://github.com/fleetdm/classified/blob/main/.github/ISSUE_TEMPLATE/%F0%9F%9A%AA-offboarding-____________.md) for the Fleetie converting to a consultant, and confirm with their manager if there is a need to retain any tools or access while they are a consultant (default to removing all access from Fleet email, and migrating to personal email for Slack and other tools unless there is a business case to retain the Fleet email and associated tool access).
 - Follow the offboarding issue for next steps, including communicating to teammates and updating equity plan.
 
+### Change a Fleetie's manager 
+
+Once the decision has been made to change a Fleetie's manager, the department head should submit a [custom issue to the BizOps team](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=&projects=&template=custom-request.md&title=Request%3A+_______________________) with the following details: 
+- Name of Fleetie 
+- Current manager 
+- New manager 
+- Effective date 
+
+Once notified, the BizOps team will: 
+- Update [Fleeties](https://docs.google.com/spreadsheets/d/1OSLn-ZCbGSjPusHPiR5dwQhheH1K8-xqyZdsOe9y7qc/edit#gid=0) to reflect the changes. 
```

Comment:
Mike: this line is the goal of this responsibility (to ensure the spread sheet is updated exactly the same way every time), so lets expand it and optionally keep line 172 and cut everything else 

The above could live on the leadership page to explain how Fleet handles such situations. 

As any current or future BizOps team member, I want to be able to follow these instructions cold to edit the spread sheet and not break anything.